.github/workflows/analyze.yml

```diff
@@ -3,54 +3,54 @@ name: Analyze
-        pip install .[all]
-    - name: Pack test recipe
-      run: |
-        pack -r cellpack/tests/recipes/v2/test_spheres.json -c cellpack/tests/packing-configs/test_config.json
-    - name: Modify JSON with PR sub_directory path
-      run: |
-        jq --arg branch_name "${{  github.ref_name  }}" '.create_report.results_output_path |= "https://cellpack-results.s3.us-west-2.amazonaws.com/\($branch_name)/spheresSST"' cellpack/tests/analysis-configs/PR_analysis_config.json > cellpack/tests/analysis-configs/PR_analysis_config_temp.json
-        mv cellpack/tests/analysis-configs/PR_analysis_config_temp.json cellpack/tests/analysis-configs/PR_analysis_config.json
-    - name: Run analysis code
-      run: analyze -a cellpack/tests/analysis-configs/PR_analysis_config.json 
-    - name: Upload results
-      uses: actions/upload-artifact@v3
-      with:
-        name: results
-        path: cellpack/tests/outputs/test_spheres/
-  Comment:
-    runs-on: ubuntu-latest
-    needs: [Analyze]
-    steps:
-    - uses: actions/checkout@v3
-    - name: Configure AWS credentials
-      uses: aws-actions/configure-aws-credentials@v1-node16
-      with:
-        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-        aws-region: us-west-2
-    - uses: actions/download-artifact@v3
-    - name: Copy files to results bucket
-      run: aws s3 cp ./results s3://cellpack-results/${{  github.ref_name  }}/ --recursive --acl public-read 
-    - uses: iterative/setup-cml@v1
-    - name: Update comment
-      env:
-          REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      run: | 
-        cml comment update --watermark-title="{workflow} report" ./results/analysis_report.md --pr
-        cat ./results/analysis_report.md
+    Analyze:
+        runs-on: ${{ matrix.os }}
+        strategy:
+            matrix:
+                python-version: [3.9]
+                os: [ubuntu-latest]
+        steps:
+            - uses: actions/checkout@v3
+            - name: Set up Python ${{ matrix.python-version }}
+              uses: actions/setup-python@v4
+              with:
+                  python-version: ${{ matrix.python-version }}
+            - name: Install Dependencies
+              run: |
+                  python -m pip install --upgrade pip
+                  pip install .[all]
+            - name: Pack test recipe
+              run: |
+                  pack -r cellpack/tests/recipes/v2/test_spheres.json -c cellpack/tests/packing-configs/test_config.json
+            - name: Modify JSON with PR sub_directory path
+              run: |
+                  jq --arg branch_name "${{  github.ref_name  }}" '.create_report.output_image_location |= "https://cellpack-results.s3.us-west-2.amazonaws.com/\($branch_name)/spheresSST/figures"' cellpack/tests/analysis-configs/PR_analysis_config.json > cellpack/tests/analysis-configs/PR_analysis_config_temp.json
```

Comment:
Looks like the url https://cellpack-results.s3.us-west-2.amazonaws.com/ leads to the cellpack-results bucket? Makes sense and looks good to me!